tests/CardErrorTest.php

```diff
@@ -24,6 +30,7 @@ public function testDecline()
             Charge::create($charge);
         } catch (Error\Card $e) {
             $this->assertSame(402, $e->getHttpStatus());
+            $this->assertStartsWith("req_", $e->getRequestId());
```

Comment:
If this just going to return a boolean, or does PHPUnit magically know that `assert*` methods should raise on false?